src/managementpartner/azext_managementpartner/managementpartner/version.py

```diff
@@ -9,4 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.1.0"
+VERSION = "0.1.1"
```

Comment:
This version seems incorrect or is it different to the version in setup.py?

Reply:
Good catch, let me fix it